### DISCUSSION TITLE
Out put of FeynCalc is not symbolic... How to solve it?

### DISCUSSION THREAD

**Discussion by JPCBPF** [FeynCalc/feyncalc#238] [Q&A]
When I run, for example:

LC[a,b,c,d] MT[a,b]

the out put is

LC[a,b,c,d] MT[a,b]

rather than 

$\varepsilon_{abcd}g^{ab}$.

The code is correct because when I do 

Contract[LC[a,b,c,d] MT[ab]]

it returns ZERO.

The same thing happens with, 

MT[a, b] MT[a, b]

which resturns to me 

MT[a, b]²

instead of 

$g_{ab}g^{ab}$.

When I do 

Contract[MT[a, b] MT[a, b]]

it returns FOUR as answer...

How can I make, for axample, 

MT[a, b] MT[a, b]

returns 

$g_{ab}g^{ab}$

instead of 

MT[a, b]²??

Thank you!!!

**Comment by vsht** [comment]
Your output format in Mathematica is set to
StandardForm instead of TraditionalForm.

You can change it for the current session by
evaluating `FCEnableTraditionalFormOutput`.

Am 12.11.23 um 23:02 schrieb JPCBPF:

**Reply by JPCBPF** [reply]
Thank you, but it did not worked...

**Comment by vsht** [comment]
Then you should try changing it to TraditionalForm globally

https://support.wolfram.com/34575?src=mathematica

Am 13.11.23 um 14:37 schrieb JPCBPF:
